outputs_quantity.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: index
+title: Quantity Outputs
+prev: supply_land.html
+next: outputs_prices.html
+gcam-version: v5.4
+---
+
+## Description of Outputs
+
+| Name | Resolution | Unit | Query | XML Tag |
+| :--- | :--- | :--- | :--- | :--- |
+| Physical Output<sup>[1](#table_footnote)</sup> | Technology, Region<sup>[2](#table_footnote)</sup>, Vintage, and Year  | Various<sup>[3](#table_footnote)</sup> | <span id="outputs by tech"><button onclick='getQuery("outputs by tech", "outputs by tech")'>outputs by tech</button></span> | `physical-output` |
+| Resource Production | Region, Resource and Year   | Various<sup>[3](#table_footnote)</sup> | <span id="resource production"><button onclick='getQuery("resource production", "resource production")'>resource production</button></span> | `output` |
+| Inputs<sup>[1](#table_footnote)</sup> | Technology, Input, Region<sup>[2](#table_footnote)</sup>, Vintage, and Year    | Various<sup>[3](#table_footnote)</sup> | <span id="inputs by tech"><button onclick='getQuery("inputs by tech", "inputs by tech")'>inputs by tech</button></span> | `demand-physical` |
```

Comment:
price-paid is also output here (in startVisitInput) and is used in several queries: elec gen costs by cooling tech, transport tech non-energy cost, food demand prices, costs by tech and input. Other things output from this function but that don't show up in the main query file are: output-unit, IO-coefficient, demand-currency, carbon-content